scripts/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "rules": {
+    "import/no-extraneous-dependencies": [2, {
+      "devDependencies": true
```

Comment:
this can even be configured in the root .eslintrc, with glob-based configuration :-D